packages/common/src/schemas/documents/default-context-schema.ts

```diff
@@ -1,30 +1,22 @@
-export const defaultContextSchema = {
-  $schema: 'http://json-schema.org/draft-07/schema#',
-  type: 'object',
-  properties: {
-    entity: {
-      type: 'object',
-      properties: {
-        type: {
-          enum: ['individual', 'business'],
-        },
-        data: {
-          type: 'object',
-          properties: {
-            additionalInfo: {
-              type: 'object',
-            },
+import { Type } from '@sinclair/typebox';
+
+export const defaultContextSchema = Type.Object({
```

Comment:
I refactored the JSON Schema into a TypeBox object so I could use `Type.Composite` in a different file. The output schema should be almost identical, but please review it.
Why it's not exactly identical? Because in some places the JSON Schema wasn't valid (`integer` instead of `number`, etc)